mentors/elixir/m.json

```
@@ -1 +1,10 @@
-[]
+[
+    {
```

```suggestion
  {
```
Should only be indented two spaces.